cicecore/cicedyn/general/ice_forcing.F90

```diff
@@ -233,8 +319,40 @@ subroutine alloc_forcing
            ocn_frc_m(nx_block,ny_block,  max_blocks,nfld,12), & ! ocn data for 12 months
         topmelt_file(ncat), &
         botmelt_file(ncat), &
-        wave_spectrum_data(nx_block,ny_block,nfreq,2,max_blocks), &
-         stat=ierr)
+        wave_spectrum_data(nx_block,ny_block,nfreq,2,max_blocks),&
+	aicen_bry(nx_block,ny_block,ncat,max_blocks), &      
```

Comment:
@SmithJos13 I am not sure how CICE handles those but it seems that even if you are not using boundary condition such as global configuration, you are still allocating these arrays. It will increase the memory consumption also for other configurations. Is it possible to connect these to a runtime configuration. So, it would be only allocated when user activates that option to provide boundary conditions)